build/ci.go

```diff
@@ -330,6 +330,7 @@ func doLint(cmdline []string) {
 	configs := []string{
 		"--vendor",
 		"--tests",
+		"--deadline=10m",
```

Comment:
Please use a shorter deadline like 2m

Reply:
Done. I hope this is enough for the `ethersphere` repo.